tests/tests/test_state_scripts.py

```diff
@@ -422,7 +440,6 @@
         "simulate_powerloss_in_commit_enter",
         {
             "RebootScripts": ["ArtifactCommit_Enter_89"],
-            "DoubleReboot": [True],
```

Comment:
If you add handling of `ArtifactRollbackReboot` (as mentioned in an earlier comment), you may be able to put this back. However, it is also pending the discussion about `verify_reboot_performed`, which I didn't quite follow.

Reply:
It won't, see reply below.

Originally I thought about having extra logic in `ArtifactRollback` to increment a counter in a `/data` log file or something to note the number of reboots and be able to verify reboots in all test cases. But I considered to be not that important: we verify that the correct state script is called via the existing log file, so no point in double verifying it for this specific state (reboot)

Reply:
You're right, this is not important, it can be removed.